addons/survey/static/src/scss/survey_views.scss

```diff
@@ -1,6 +1,39 @@
 .o_kanban_card_survey {
+    min-height: 100px;
+    
     .certification_icon {
-        font-size: 20px;
+        overflow: hidden;
```

Comment:
Waaaaaaay too much CSS here for a simple kanban card. Why do you need so much custom css ?